src/tribol/interface/tribol.cpp

```diff
@@ -496,6 +626,18 @@ void registerNodalResponse( integer meshId,
 
 } // end registerNodalResponse()
 
+//------------------------------------------------------------------------------
+mfem::ParGridFunction getMfemResponse( integer cs_id )
+{
+   auto coupling_scheme = CouplingSchemeManager::getInstance().getCoupling(cs_id);
+   SLIC_ERROR_ROOT_IF(
+      !coupling_scheme->hasMfemData(), 
+      "Coupling scheme does not contain MFEM data. "
```

Comment:
See comment above about error statement and updating function name.

Reply:
Fixed.